velox/functions/prestosql/tests/ComparisonsTest.cpp

```diff
@@ -153,6 +153,57 @@ TEST_F(ComparisonsTest, eqDecimal) {
       " SHORT_DECIMAL(10,4))");
 }
 
+TEST_F(ComparisonsTest, gtLtDecimal) {
+  auto runAndCompare = [&](std::string expr,
+                           std::vector<VectorPtr>& inputs,
+                           VectorPtr expectedResult) {
+    auto actual = evaluate<SimpleVector<bool>>(expr, makeRowVector(inputs));
+    test::assertEqualVectors(actual, expectedResult);
+  };
+
+  // Short Decimals test.
+  std::vector<VectorPtr> inputs = {
+      makeNullableShortDecimalFlatVector(
+          {1, std::nullopt, 3, -3, std::nullopt, 4}, DECIMAL(10, 5)),
+      makeNullableShortDecimalFlatVector(
+          {0, 2, 3, -5, std::nullopt, 5}, DECIMAL(10, 5))};
+  auto expected = makeNullableFlatVector<bool>(
+      {true, std::nullopt, false, true, std::nullopt, false});
+  auto expected2 = makeNullableFlatVector<bool>(
+      {true, std::nullopt, true, true, std::nullopt, false});
+
+  runAndCompare("c0 > c1", inputs, expected);
+  runAndCompare("c1 < c0", inputs, expected);
+  // Gte/Lte
+  runAndCompare("c0 >= c1", inputs, expected2);
+  runAndCompare("c1 <= c0", inputs, expected2);
+
+  // Long Decimals test.
+  inputs = {
```

Comment:
make this `longDecimalInputs` and move it on top below `ShortDecimalInputs` so that it is easier to review against the expected results.